doc/users/next_whats_new/colorbar_minor_ticks.rst

```diff
@@ -0,0 +1,10 @@
+Add ``minorticks_on()/off()`` methods for colorbar
+--------------------------------------------------
+
+A new method :meth:`~matplotlib.colorbar.Colobar.minorticks_on` is
```

Comment:
You could use `` `.Colorbar.minorticks_on` ``

Reply:
thanks... I wasn't really familiar with this style of documentation so I was just following what I saw in some of the other whats new files :D